{"uid":"51963d24929c7184","name":"test_setup","fullName":"tests.integration.test_smoke#test_setup","historyId":"d623373ae070a62cab8e4b2f9dd522ae","time":{"start":1748481856144,"stop":1748482179519,"duration":323375},"status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: sysbench/0","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 898, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/tests/integration/test_smoke.py\", line 33, in test_setup\n    await ops_test.model.wait_for_idle(\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3245, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3136, in _raise_for_status\n    raise error_type(\njuju.errors.JujuUnitError: Unit in error: sysbench/0","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1748481855928,"stop":1748481855928,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1748481855929,"stop":1748481856143,"duration":214},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1748481855929,"stop":1748481855929,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1748481856144,"stop":1748481856144,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: sysbench/0","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n                             ^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 898, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/tests/integration/test_smoke.py\", line 33, in test_setup\n    await ops_test.model.wait_for_idle(\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3245, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/work/postgresql-k8s-bundle/postgresql-k8s-bundle/.tox/integration/lib/python3.12/site-packages/juju/model/__init__.py\", line 3136, in _raise_for_status\n    raise error_type(\njuju.errors.JujuUnitError: Unit in error: sysbench/0","steps":[],"attachments":[{"uid":"5cf3c0b999361d43","name":"log","source":"5cf3c0b999361d43.txt","type":"text/plain","size":19347}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1748482179529,"stop":1748482179529,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1748482179533,"stop":1748482180243,"duration":710},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1748482180244,"stop":1748482180244,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1748482180244,"stop":1748482180244,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1748482180244,"stop":1748482180244,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_smoke"},{"name":"host","value":"fv-az1288-828"},{"name":"thread","value":"16769-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_smoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"bb835e38bd61bb07","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":27,"skipped":0,"passed":298,"unknown":9,"total":334},"items":[{"uid":"ede48f777ae3abc6","reportUrl":"../2104//#testresult/ede48f777ae3abc6","status":"passed","time":{"start":1748395427937,"stop":1748396056505,"duration":628568}},{"uid":"e185bcdd873ae10","reportUrl":"../2102//#testresult/e185bcdd873ae10","status":"broken","statusDetails":"juju.errors.JujuError: refreshing charm \"ch:amd64/jammy/self-signed-certificates-304\": Post \"https://api.charmhub.io/v2/charms/refresh\": read tcp 10.1.235.133:40502->185.125.188.58:443: read: connection timed out","time":{"start":1748309088148,"stop":1748310134569,"duration":1046421}},{"uid":"a609a15c41315ed0","reportUrl":"../2101//#testresult/a609a15c41315ed0","status":"passed","time":{"start":1748222647320,"stop":1748223266472,"duration":619152}},{"uid":"6dabf872cda644e4","reportUrl":"../2100//#testresult/6dabf872cda644e4","status":"passed","time":{"start":1748136234052,"stop":1748136879004,"duration":644952}},{"uid":"246c71401e2fbff3","reportUrl":"../2099//#testresult/246c71401e2fbff3","status":"passed","time":{"start":1748049829701,"stop":1748050407217,"duration":577516}},{"uid":"fdec6d7b4e0bd78d","reportUrl":"../2098//#testresult/fdec6d7b4e0bd78d","status":"passed","time":{"start":1747963424894,"stop":1747964076171,"duration":651277}},{"uid":"fb512c2f85c0de09","reportUrl":"../2097//#testresult/fb512c2f85c0de09","status":"passed","time":{"start":1747877109145,"stop":1747877839212,"duration":730067}},{"uid":"188c888e1c798592","reportUrl":"../2095//#testresult/188c888e1c798592","status":"passed","time":{"start":1747790684213,"stop":1747791427801,"duration":743588}},{"uid":"43b28e0c668ce36b","reportUrl":"../2092//#testresult/43b28e0c668ce36b","status":"passed","time":{"start":1747704285812,"stop":1747704962894,"duration":677082}},{"uid":"700215600b96c842","reportUrl":"../2090//#testresult/700215600b96c842","status":"passed","time":{"start":1747617839023,"stop":1747618452406,"duration":613383}},{"uid":"ecb6da211b2e2f48","reportUrl":"../2088//#testresult/ecb6da211b2e2f48","status":"passed","time":{"start":1747531432422,"stop":1747532069630,"duration":637208}},{"uid":"a4ac6e36f566a3e","reportUrl":"../2086//#testresult/a4ac6e36f566a3e","status":"passed","time":{"start":1747445024668,"stop":1747445616779,"duration":592111}},{"uid":"c225cd29f91805bb","reportUrl":"../2084//#testresult/c225cd29f91805bb","status":"passed","time":{"start":1747358696169,"stop":1747359331487,"duration":635318}},{"uid":"4e8bc35d4c5e8b80","reportUrl":"../2082//#testresult/4e8bc35d4c5e8b80","status":"passed","time":{"start":1747272298186,"stop":1747272940119,"duration":641933}},{"uid":"fd9566d0e8d8ae6d","reportUrl":"../2080//#testresult/fd9566d0e8d8ae6d","status":"passed","time":{"start":1747185819937,"stop":1747186523133,"duration":703196}},{"uid":"e1b295237925f962","reportUrl":"../2077//#testresult/e1b295237925f962","status":"passed","time":{"start":1747099421506,"stop":1747100081493,"duration":659987}},{"uid":"5550ad6cfb0f8f8e","reportUrl":"../2076//#testresult/5550ad6cfb0f8f8e","status":"passed","time":{"start":1747013018429,"stop":1747013646165,"duration":627736}},{"uid":"e401b440c0e35bc3","reportUrl":"../2075//#testresult/e401b440c0e35bc3","status":"passed","time":{"start":1746926703798,"stop":1746927338196,"duration":634398}},{"uid":"4d095ae733d35827","reportUrl":"../2074//#testresult/4d095ae733d35827","status":"passed","time":{"start":1746840209495,"stop":1746840948077,"duration":738582}},{"uid":"5cba8b6716a00ca5","reportUrl":"../2073//#testresult/5cba8b6716a00ca5","status":"passed","time":{"start":1746753796709,"stop":1746754416337,"duration":619628}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"51963d24929c7184.json","parameterValues":[]}